signer/src/blocklist_client.rs

```diff
@@ -44,11 +44,19 @@ impl BlocklistClient {
     pub fn new(ctx: &impl Context) -> Option<Self> {
         let config = ctx.config().blocklist_client.as_ref()?;
 
-        let config = Configuration {
+        let mut config = Configuration {
```

Comment:
I know this isn't your doing, but it feels pretty extreme that we're talking a whole `Context` to build this when all we really need is the url -- makes the tests below a lot more complicado than they need to be 😬

Reply:
Yeah, I will fix this later